ted_sws/__init__.py

```diff
@@ -68,6 +68,18 @@ def MONGO_DB_PORT(self) -> int:
     def MONGO_DB_AGGREGATES_DATABASE_NAME(self) -> str:
         return EnvConfigResolver().config_resolve()
 
+    @property
+    def AIRFLOW_POSTGRES_DB_NAME(self) -> str:
+        return EnvConfigResolver().config_resolve()
+
+    @property
+    def AIRFLOW_POSTGRES_PASSWORD(self) -> str:
```

Comment:
why this configuration is in MongoDBConfig ?

Reply:
I deleted this property because is not necessary.